lib/schema_matching.js

```diff
@@ -0,0 +1,331 @@
+const { match } = require('assert')
+const fs = require('fs')
+const source_files = fs.readdirSync('../sources')
+
+/**
+ * This function scrapes all crosswalk data from the 'sources' directory
+ * and turns it into an array with all crosswalks.
+ * 
+ * @returns {Array} All of the crosswalk data
+ */
+function getCrosswalkData() {
+    const scraped_crosswalks = []
+    for (const source_file of source_files) {
+        const source_data = require('../sources'+'/'+source_file)
+        for (const entry of source_data) {
```

Comment:
Are there any chances source_data or source_files could be None resulting in a error? I am not sure, but wanted to ask.

Reply:
I don't think so because the scraper is just reading the data from each of the files in the sources folder, which should always have data in it